### ISSUE TITLE
[Max] Bump / Normal Maps Support Clarification

### ISSUE THREAD

**Issue by EtagiBI** [BabylonJS/Exporters#441]
In documentation it says "The bump map (or normal map) and its weight are used.". When I export a model with Bump map, I get an ugly metallic cover on my model. When I export a model with Normal map, I get the error: "Texture type is not supported. Use a Bitmap instead.". That's confusing =/. What should I do to keep bump effect after export?

It would be nice to know a correct node setup for normal maps if they're supported. Standard scheme "Bitmap -> Normal Bump -> Material" doesn't work with exporter.

Test models:
[https://github.com/BabylonJS/Exporters/files/2693439/CUBES.zip](url)

**Comment by kcoley** [comment]
Hi @EtagiBI . You should be able to attach a bitmap node referencing a normal map to the bump map input of the Physical Material:
![image](https://user-images.githubusercontent.com/3946928/51815502-2ee8ba00-2276-11e9-9c81-3b4941563cb3.png)

Here is a sample project: 
[NormalMapTest.zip](https://github.com/BabylonJS/Exporters/files/2801495/NormalMapTest.zip)

Are you trying to use a normal map or a height map, and what format is your texture in?  Also, which material are you trying to use?

The test model link you provided seems to not be valid for me.

**Comment by EtagiBI** [comment]
Hi,

> You should be able to attach a bitmap node referencing a normal map to the bump map input of the Physical Material:

Hmm. I always use Standard Material, because 99% of 3DS Max models we get from our contributors have Standard materials. Do you mean that Normal maps're incompatible with Standard material?..

Here're the models again:
[CUBES.zip](https://github.com/BabylonJS/Exporters/files/2802085/CUBES.zip)

**Comment by kcoley** [comment]
> Do you mean that Normal maps're incompatible with Standard material?

@EtagiBI thanks for the sample model.   Standard Materials work as well.  If you plan on exporting to glTF, the exporter has to do some value conversions to try to approximate a physically-based material, which glTF expects, so the resulting files may not look exactly the same as authored.  

> When I export a model with Normal map, I get the error: "Texture type is not supported. Use a Bitmap instead.". That's confusing =/. 

Currently, the exporter does not handle the Normal Bump node.  The error in the log is misleading since it is referring to the Normal Bump node, not the format of the texture.  If you were to plug the bitmap texture directly into the standard material bump input, it would export without the error.  

> When I export a model with Bump map, I get an ugly metallic cover on my model

I did notice that the normal map you are using does not seem to be a tangent-space normal map, which is what glTF is expecting:
![normal](https://user-images.githubusercontent.com/3946928/51865282-ad873b00-22fa-11e9-9bd4-e3617df935c9.jpg)

I used a tool called [Crazy Bump](http://www.crazybump.com/) to convert the bump map to a tangent-space normal map:
![bump_to_normal](https://user-images.githubusercontent.com/3946928/51865338-cf80bd80-22fa-11e9-9d39-73ae7f620a9d.jpg)

I noticed that you plugged in a height map to the other standard material into the bump input.  glTF would expect that to be a tangent-space normal map.  

Are you trying to target glTF or the babylon file format?
